plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGeneratorSitesWithoutData.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div id="javascript-text">
+    <pre v-select-on-focus="{}" class="codeblock" v-text="trackingCode" ref="trackingCode"/>
+  </div>
+  <JsTrackingCodeAdvanceOptions
```

Comment:
I think it's in multiple places as well.. should be advance**d** options
```suggestion
  <JsTrackingCodeAdvancedOptions
```

Reply:
@michalkleiner updated